cloudesire-api-client/src/main/java/com/cloudesire/platform/apiclient/api/UserApi.java

```diff
@@ -56,28 +56,25 @@
     Call<MyUserDTO> create( @Body MyUserDTO userDTO );
 
     @DELETE( "user/{id}/payment" )
-    Call<Void> deletePaymentDataForAdmin( @Path( "id" ) Integer id );
+    Call<Void> deletePaymentDataForAdmin( @Path( "id" ) int id );
 
     @DELETE( "user/payment" )
     Call<Void> deletePaymentData();
 
     @DELETE( "user/{id}" )
-    Call<Void> delete( @Path( "id" ) Integer id );
+    Call<Void> delete( @Path( "id" ) int id );
 
     @PATCH( "user/{id}" )
-    Call<Void> partialUpdate( @Path( "id" ) Integer id, @Body Map<String, Object> input );
+    Call<Void> partialUpdate( @Path( "id" ) int id, @Body Map<String, Object> input );
 
     @PATCH( "user/{id}" )
-    Call<Void> partialUpdate( @Path( "id" ) Integer id, @Body Map<String, Object> input, @Query( "language" ) String language );
+    Call<Void> partialUpdate( @Path( "id" ) int id, @Body Map<String, Object> input, @Query( "language" ) String language );
 
     @GET( "user" )
     Call<List<MyUserDTO>> getAll( @Query( "textField" ) String textField, @Query( "companyId" ) Integer companyId,
             @Query( "role" ) UserRole role, @Query( "disabled" ) Boolean disabled,
             @QueryMap Map<String, String> pageRequest );
 
-    @GET( "user/{id}/reseller" )
-    Call<ResellerDTO> getResellerProfile( @Path( "id" ) Integer id );
```

Comment:
dupe of https://github.com/ClouDesire/java-api-client/pull/125/commits/f7f1bd275b158af78d94ca00e5b26022bd863203#diff-0761a8e1f7bc1edeab70b58b73d4c464R126